packages/vscode-extension/src/commonlib/graphLogin.ts

```diff
@@ -159,6 +159,7 @@ export class GraphLogin extends login implements GraphTokenProvider {
   }
 
   async getStatus(): Promise<LoginStatus> {
+    await GraphLogin.codeFlowInstance.reloadCache();
```

Comment:
Reminder - CLI may have the same issue...

Reply:
Good point. Let me update it in later PR.